<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ynohW8-64b-one" name="HBTestesNG.TestCases.Test_VoiceOACD_ByAbandon_Transfer_WrapTimeZero" tests="14" failures="6" timestamp="17 Oct 2017 19:52:46 GMT" time="1892.566" errors="0">
  <testcase name="VoiceOACD_ByAbandon_CTransfer_ToIRN_SameGroup" time="0.001" classname="HBTestesNG.TestCases.Test_VoiceOACD_ByAbandon_Transfer_WrapTimeZero"/>
  <testcase name="VoiceOACD_ByAbandon_BTransfer_ToIRN_SameGroup" time="0.000" classname="HBTestesNG.TestCases.Test_VoiceOACD_ByAbandon_Transfer_WrapTimeZero"/>
  <testcase name="VoiceOACD_ByAbandon_CTransfer_ToExt_SameGroup_byName" time="159.240" classname="HBTestesNG.TestCases.Test_VoiceOACD_ByAbandon_Transfer_WrapTimeZero"/>
  <testcase name="VoiceOACD_ByAbandon_CTransfer_ToExt_DifferentGroup" time="165.393" classname="HBTestesNG.TestCases.Test_VoiceOACD_ByAbandon_Transfer_WrapTimeZero">
    <failure type="java.lang.AssertionError" message="VoiceOACD_ByAbandon-&gt;Ans-&gt;Consult Transfer to Ext on Different Group has failed because =&gt; [agent2yc1@c1yc1.com: AIC shows idle, but it still says &quot; Stop taking reauests &quot;] expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: VoiceOACD_ByAbandon->Ans->Consult Transfer to Ext on Different Group has failed because => [agent2yc1@c1yc1.com: AIC shows idle, but it still says " Stop taking reauests "] expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at HBTestesNG.BaseObjects.TestCaseObject.failTest(TestCaseObject.java:464)
	at HBTestesNG.BaseObjects.TestCaseObject.endTestCase(TestCaseObject.java:243)
	at HBTestesNG.TestCases.Test_VoiceOACD_ByAbandon_Transfer_WrapTimeZero.VoiceOACD_ByAbandon_CTransfer_ToExt_DifferentGroup(Test_VoiceOACD_ByAbandon_Transfer_WrapTimeZero.java:457)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- VoiceOACD_ByAbandon_CTransfer_ToExt_DifferentGroup -->
  <testcase name="VoiceOACD_ByAbandon_BTransfer_ToExt_SameGroup_byName" time="152.260" classname="HBTestesNG.TestCases.Test_VoiceOACD_ByAbandon_Transfer_WrapTimeZero"/>
  <testcase name="VoiceOACD_ByAbandon_BTransfer_ToExternal" time="157.149" classname="HBTestesNG.TestCases.Test_VoiceOACD_ByAbandon_Transfer_WrapTimeZero"/>
  <testcase name="VoiceOACD_ByAbandon_BTransfer_ToExt_DifferentGroup" time="157.217" classname="HBTestesNG.TestCases.Test_VoiceOACD_ByAbandon_Transfer_WrapTimeZero">
    <failure type="java.lang.AssertionError" message="VoiceOACD_ByAbandon-&gt;Ans-&gt;Blind Transfer to Ext on Different Group has failed because =&gt; [agent2yc1@c1yc1.com: AIC shows idle, but it still says &quot; Stop taking reauests &quot;] expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: VoiceOACD_ByAbandon->Ans->Blind Transfer to Ext on Different Group has failed because => [agent2yc1@c1yc1.com: AIC shows idle, but it still says " Stop taking reauests "] expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at HBTestesNG.BaseObjects.TestCaseObject.failTest(TestCaseObject.java:464)
	at HBTestesNG.BaseObjects.TestCaseObject.endTestCase(TestCaseObject.java:243)
	at HBTestesNG.TestCases.Test_VoiceOACD_ByAbandon_Transfer_WrapTimeZero.VoiceOACD_ByAbandon_BTransfer_ToExt_DifferentGroup(Test_VoiceOACD_ByAbandon_Transfer_WrapTimeZero.java:137)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- VoiceOACD_ByAbandon_BTransfer_ToExt_DifferentGroup -->
  <testcase name="VoiceOACD_ByAbandon_BTransfer_ToIRN_DifferentGroup" time="147.186" classname="HBTestesNG.TestCases.Test_VoiceOACD_ByAbandon_Transfer_WrapTimeZero">
    <failure type="java.lang.AssertionError" message="VoiceOACD_ByAbandon-&gt;Ans-&gt;Blind Transfer to IRN on Different Group_WrapTimeZero has failed because =&gt; [agent2yc1@c1yc1.com: AIC shows idle, but it still says &quot; Stop taking reauests &quot;] expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: VoiceOACD_ByAbandon->Ans->Blind Transfer to IRN on Different Group_WrapTimeZero has failed because => [agent2yc1@c1yc1.com: AIC shows idle, but it still says " Stop taking reauests "] expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at HBTestesNG.BaseObjects.TestCaseObject.failTest(TestCaseObject.java:464)
	at HBTestesNG.BaseObjects.TestCaseObject.endTestCase(TestCaseObject.java:243)
	at HBTestesNG.TestCases.Test_VoiceOACD_ByAbandon_Transfer_WrapTimeZero.VoiceOACD_ByAbandon_BTransfer_ToIRN_DifferentGroup(Test_VoiceOACD_ByAbandon_Transfer_WrapTimeZero.java:275)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- VoiceOACD_ByAbandon_BTransfer_ToIRN_DifferentGroup -->
  <testcase name="VoiceOACD_ByAbandon_BTransfer_ToExt_SameGroup" time="152.402" classname="HBTestesNG.TestCases.Test_VoiceOACD_ByAbandon_Transfer_WrapTimeZero"/>
  <testcase name="VoiceOACD_ByAbandon_CTransfer_ToExt_SameGroup" time="160.215" classname="HBTestesNG.TestCases.Test_VoiceOACD_ByAbandon_Transfer_WrapTimeZero"/>
  <testcase name="VoiceOACD_ByAbandon_CTransfer_ToExternal" time="165.073" classname="HBTestesNG.TestCases.Test_VoiceOACD_ByAbandon_Transfer_WrapTimeZero"/>
  <testcase name="VoiceOACD_ByAbandon_CTransfer_ToIRN_DifferentGroup" time="154.613" classname="HBTestesNG.TestCases.Test_VoiceOACD_ByAbandon_Transfer_WrapTimeZero">
    <failure type="java.lang.AssertionError" message="VoiceOACD_ByAbandon-&gt;Ans-&gt;Consult Transfer to IRN on Different Group_WrapTimeZero has failed because =&gt; [agent2yc1@c1yc1.com: AIC shows idle, but it still says &quot; Stop taking reauests &quot;] expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: VoiceOACD_ByAbandon->Ans->Consult Transfer to IRN on Different Group_WrapTimeZero has failed because => [agent2yc1@c1yc1.com: AIC shows idle, but it still says " Stop taking reauests "] expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at HBTestesNG.BaseObjects.TestCaseObject.failTest(TestCaseObject.java:464)
	at HBTestesNG.BaseObjects.TestCaseObject.endTestCase(TestCaseObject.java:243)
	at HBTestesNG.TestCases.Test_VoiceOACD_ByAbandon_Transfer_WrapTimeZero.VoiceOACD_ByAbandon_CTransfer_ToIRN_DifferentGroup(Test_VoiceOACD_ByAbandon_Transfer_WrapTimeZero.java:597)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- VoiceOACD_ByAbandon_CTransfer_ToIRN_DifferentGroup -->
  <testcase name="VoiceOACD_ByAbandon_BTransfer_ToExt_DifferentGroup_byName" time="157.252" classname="HBTestesNG.TestCases.Test_VoiceOACD_ByAbandon_Transfer_WrapTimeZero">
    <failure type="java.lang.AssertionError" message="VoiceOACD_ByAbandon-&gt;Ans-&gt;Blind Transfer to Ext on Different Group_byName has failed because =&gt; [agent2yc1@c1yc1.com: AIC shows idle, but it still says &quot; Stop taking reauests &quot;] expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: VoiceOACD_ByAbandon->Ans->Blind Transfer to Ext on Different Group_byName has failed because => [agent2yc1@c1yc1.com: AIC shows idle, but it still says " Stop taking reauests "] expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at HBTestesNG.BaseObjects.TestCaseObject.failTest(TestCaseObject.java:464)
	at HBTestesNG.BaseObjects.TestCaseObject.endTestCase(TestCaseObject.java:243)
	at HBTestesNG.TestCases.Test_VoiceOACD_ByAbandon_Transfer_WrapTimeZero.VoiceOACD_ByAbandon_BTransfer_ToExt_DifferentGroup_byName(Test_VoiceOACD_ByAbandon_Transfer_WrapTimeZero.java:321)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- VoiceOACD_ByAbandon_BTransfer_ToExt_DifferentGroup_byName -->
  <testcase name="VoiceOACD_ByAbandon_CTransfer_ToExt_DifferentGroup_byName" time="164.565" classname="HBTestesNG.TestCases.Test_VoiceOACD_ByAbandon_Transfer_WrapTimeZero">
    <failure type="java.lang.AssertionError" message="VoiceOACD_ByAbandon-&gt;Ans-&gt;Consult Transfer to Ext on Different Group_byName has failed because =&gt; [agent2yc1@c1yc1.com: AIC shows idle, but it still says &quot; Stop taking reauests &quot;] expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: VoiceOACD_ByAbandon->Ans->Consult Transfer to Ext on Different Group_byName has failed because => [agent2yc1@c1yc1.com: AIC shows idle, but it still says " Stop taking reauests "] expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at HBTestesNG.BaseObjects.TestCaseObject.failTest(TestCaseObject.java:464)
	at HBTestesNG.BaseObjects.TestCaseObject.endTestCase(TestCaseObject.java:243)
	at HBTestesNG.TestCases.Test_VoiceOACD_ByAbandon_Transfer_WrapTimeZero.VoiceOACD_ByAbandon_CTransfer_ToExt_DifferentGroup_byName(Test_VoiceOACD_ByAbandon_Transfer_WrapTimeZero.java:644)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- VoiceOACD_ByAbandon_CTransfer_ToExt_DifferentGroup_byName -->
</testsuite> <!-- HBTestesNG.TestCases.Test_VoiceOACD_ByAbandon_Transfer_WrapTimeZero -->
